test/lib/ex_debug_toolbar/logger_test.exs

```diff
@@ -0,0 +1,25 @@
+defmodule ExDebugToolbar.LoggerTest do
+  use ExUnit.Case, async: true
```

Comment:
you are modifying env in this test. It has to be `async: false`

Reply:
this var no effect on other tests, but you're right, it should be changed to avoid surprises in the future